<launch>
	<!-- Set parameters 115200 57600 19200 9600-->
	<param name="port" value="/dev/stibase_motor" />
	<param name="baudrate" value="57600" />
	<param name="id" value="1" />
	<param name="maxRPM" value="3800" />
	<param name="topicSafety" value="/safety" />
	<param name="topicSub" value="/driver1_query" />
	<param name="topicPub" value="/driver1_respond" />
	<param name="revert" value="0" />
	<!-- <param name="publish_tag_detections_image" type="bool" value="true" />  -->
	<node pkg="base_pkg" type="driverControl.py" name="driverLeft_node" output="screen"> </node>
</launch>